sdk/eventhub/azure-eventhubs/api_review.py

```diff
@@ -0,0 +1,235 @@
+
+class EventHubProducerClient(ClientBaseAsync):
+
+    eh_name  # type: str
```

Comment:
event_hub_name? Did we shorten it for a particular reason?

Reply:
yes, we'll change to eventhub_name and fully_qualified_namespace